.github/workflows/timhwang_yarn_test.yml

```
@@ -0,0 +1,14 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Run TypeScript tests
+      run: |
+        cd timhwang21
+        yarn
```

```suggestion
        yarn --frozen-lockfile
```

Minor -- this is generally the recommendation in CI to prevent unexpected dependency changes!